jinja2/filters.py

```diff
@@ -84,6 +84,44 @@ def attrgetter(item):
     return attrgetter
 
 
+def make_multi_attrgetter(environment, attribute, postprocess=None):
+    """Returns a callable that looks up the given comma separated
+    attributes from a passed object with the rules of the environment.
+    Dots are allowed to access attributes of each attribute.  Integer
+    parts in paths are looked up as integers.
+
+    The value returned by the returned callable is a list of extracted
+    attribute values.
+
+    Examples of attribute: "attr1,attr2", "attr1.inner1.0,attr2.inner2.0", etc.
+    """
+    def _prepare_attribute_parts(attr):
```

Comment:
I see that this logic is basically the same logic from `make_attrgetter`. Would it be possible to abstract this out and use it in both places?

That way if we need to adjust this in the future, we won't have to remember to do it in both places.

Reply:
Sure, will do

Reply:
Will do

Reply:
@kevin-brown pushed a fix.